connector/manifest.xml

```diff
@@ -10,24 +10,39 @@
     <vendor name="vendor"/>
     <driver name="driver"/>
     <customizations>
-      <customization name="CAP_SELECT_INTO" value="yes"/>
-      <customization name="CAP_SELECT_TOP_INTO" value="yes"/>
+      <!-- Metadata -->
+      <customization name="CAP_FAST_METADATA" value="yes"/>
```

Comment:
question: What's our approach here when we want to keep the default value? Do we explicitly set to the default, or are we just omitting it?

I see that we are explicitly setting the default for `CAP_FAST_METADATA`, but implicitly using the default for `CAP_QUERY_TOP_0_METADATA` by omission. Unless there's a reason to do otherwise, I'd suggest that we take a consistent approach across all capabilities

Reply:
Made the change to omit the option where I was setting the values that were the same as the default.  I tested it and confirmed the behavior was the same.  
I was following the document too literally where it said "set 'yes' if..." even where 'yes' was the default.  Things are consistent now.